### DISCUSSION TITLE
API to interrogate request and response logs

### DISCUSSION THREAD

**Discussion by ejt-landmark** [mockoon/mockoon#583] [Ideas] [answered]
It would be great to have an API to verify the payload for the requests being sent to Mockoon endpoints. This would allow you to ensure your code is sending a request to the endpoint and verify that the expected payload to the mock endpoint is being received.

**Comment by 255kb** [comment, marked as answer]
Hi,
This is a good idea and would be a nice feature. Feel free to open an issue so we can track this. 🙂
